inference-engine/src/inference_engine/include/openvino/runtime/remote_context.hpp

```diff
@@ -136,6 +136,14 @@ class OPENVINO_RUNTIME_API RemoteContext {
      * @return A map of name/parameter elements.
      */
     ParamMap get_params() const;
+
+    /**
+     * @brief This function is used to create host tensor object friendly for current device
```

Comment:
please, add note that it's device friendly, we can also add an example that it's USM host for GPU